oneshot.d/browser-update-default-data

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+MOZEMBED_DIR=/home/$USER/.mozilla/mozembed
+BROWSER_DATA_DIR=/usr/share/sailfish-browser/data
+
+if [ ! -d "$MOZEMBED_DIR" ]; then
+    mkdir -p $MOZEMBED_DIR
+fi
+
+if [ ! -f "$MOZEMBED_DIR/ua-update.json" ]; then
+    while IFS='' read -r line || [[ -n "$line" ]]; do
+        # Skip comment lines
+        if [[ $line !=  *\/\/* ]] ; then
+            echo "$line" >> "$MOZEMBED_DIR/ua-update.json"
+        fi
+    done < "$BROWSER_DATA_DIR/ua-update.json.in"
```

Comment:
Comments at the end of the lines are not allowed.

Yes, grep works just fine. As it is a bit simpler will change to that.